dracut/30ignition/ignition-sysusers.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Ignition (sysusers)
+Documentation=https://github.com/coreos/ignition
+ConditionPathExists=/etc/initrd-release
+DefaultDependencies=false
+Before=ignition-complete.target
+
+OnFailure=emergency.target
+OnFailureJobMode=isolate
+
+# Stage order: setup -> fetch-offline [-> fetch] -> disks -> mount -> sysusers -> files.
+After=ignition-mount.service
+Before=ignition-files.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+EnvironmentFile=/run/ignition.env
+ExecStart=/usr/sbin/chroot /sysroot /bin/bash -c "mount proc /proc -t proc && /usr/bin/systemd-sysusers && umount /proc"
```

Comment:
A few doubts I got while going through this:
 * I think the on-disk results of this will need to be wired into existing selinux-relabeling signaling mechanisms, right?
 * should this logic be better placed in a script file or in Go as part of Ignition?
 * should sysusers and config-users handling be moved to a single dedicated Ignition stage?
 * naming bikeshedding?

Reply:
Instead of literally `chroot`ing, we should lift https://github.com/openshift/os/blob/e808d5f364aae2a846793a0d7af7be8cc886871d/overlay.d/05rhcos/usr/lib/dracut/modules.d/40rhcos-fips/rhcos-fips.sh#L97-L120 out of there and into e.g. a `sysroot-bwrap` command we can just call.